Core.UnitTests/MutableReflection/Implementation/MemberSelectorTest.cs

```diff
@@ -227,18 +235,18 @@ public void SelectSingleType ()
     {
       var types = new[]
                   {
-                      CreateTypeStub ("type1", TypeAttributes.NestedAssembly),
```

Comment:
Since we're now working with actual custom made types and not with simple mocks I needed to pass valid type attributes to avoid exceptions like

```
System.ArgumentException : Bad type attributes. Nested visibility flag set on a non-nested type.
```

Seeing as this is testing the filtering based on the value `_bindingFlagsEvaluatorMock` returns, the actual flags are not important for the test and I changed them to something valid.